.github/actions/setup/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Setup
+description: Setup Node.js and install dependencies
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
```

Comment:
18.x is not LTS, worth changing?

Reply:
@thymikee in react navigation repo we had some issues with Node 18 and SSL changes so didn't try in this PR. will check if we can upgrade after this one is merged